--- env/lib/python3.4/site-packages/bitcoinrpc/util.py	2014-07-15 20:35:08.000000000 +1000
+++ env/lib/python3.3/site-packages/bitcoinrpc/util.py	2014-03-19 01:27:03.102535632 +1100
@@ -35,7 +35,7 @@
         if len(args_t) > len(self._fields):
             raise TypeError("Number of arguments is larger than of predefined fields")
         # Copy default values
-        for (k, v) in self._defaults.iteritems():
+        for (k, v) in self._defaults.items():
             self.__dict__[k] = copy(v)
         # Set pass by value arguments
         self.__dict__.update(zip(self._fields, args_t))
@@ -46,4 +46,4 @@
         return '{module}.{classname}({slots})'.format(
             module=self.__class__.__module__, classname=self.__class__.__name__,
             slots=", ".join('{k}={v!r}'.format(k=k, v=v) for k, v in
-                            self.__dict__.iteritems()))
+                            self.__dict__.items()))
